packages/typescript-estree/README.md

```diff
@@ -215,7 +216,7 @@ interface ParseAndGenerateServicesOptions extends ParseOptions {
    * This means that if the parser encounters a file not included in any of the provided `project`s,
    * it will not error, but will instead parse the file and its dependencies in a new program.
    */
-  createDefaultProgram?: boolean;
+  deprecated__createDefaultProgram?: boolean;
```

Comment:
Personally I like the all-caps to make it more obnoxious and obvious
```suggestion
  DEPRECATED__createDefaultProgram?: boolean;
```

Or could go the react route 😆 

```suggestion
  DEPRECATED_DO_NOT_USE_OR_YOU_WILL_BE_FIRED__createDefaultProgram?: boolean;
```